war/src/test/java/io/lumeer/remote/rest/OrganizationServiceIntegrationTest.java

```diff
@@ -0,0 +1,233 @@
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -----------------------------------------------------------------------/
+ */
+package io.lumeer.remote.rest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import io.lumeer.api.dto.JsonOrganization;
+import io.lumeer.api.model.Organization;
+import io.lumeer.api.model.Permission;
+import io.lumeer.api.model.Permissions;
+import io.lumeer.api.model.Resource;
+import io.lumeer.api.model.Role;
+import io.lumeer.core.AuthenticatedUser;
+import io.lumeer.core.facade.OrganizationFacade;
+import io.lumeer.core.model.SimplePermission;
+import io.lumeer.engine.IntegrationTestBase;
+import io.lumeer.storage.api.dao.UserDao;
+import io.lumeer.storage.api.exception.ResourceNotFoundException;
+import io.lumeer.storage.mongodb.model.MongoUser;
+
+import org.assertj.core.api.SoftAssertions;
+import org.jboss.arquillian.junit.Arquillian;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+import javax.inject.Inject;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.GenericType;
+import javax.ws.rs.core.Link;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.UriBuilder;
+
+@RunWith(Arquillian.class)
+public class OrganizationServiceIntegrationTest extends IntegrationTestBase {
+
+   private static final String USER = AuthenticatedUser.DEFAULT_EMAIL;
+
+   private static final String CODE1 = "TORG";
+   private static final String CODE2 = "TORG2";
+   private static final String NAME = "Testing organization";
+   private static final String COLOR = "#ff0000";
+   private static final String ICON = "fa-search";
+
+   private static final Set<Role> USER_ROLES = Organization.ROLES;
+
+   private static final Permission USER_PERMISSION = new SimplePermission(USER, USER_ROLES);
+
+   private static final String SERVER_URL = "http://localhost:8080";
+   private static final String ORGANIZATION_PATH = "/" + PATH_CONTEXT + "/rest/" + "organizations";
+   private static final String ORGANIZATION_URL = SERVER_URL + ORGANIZATION_PATH;
+
+   @Inject
+   private OrganizationFacade organizationFacade;
+
+   @Inject
+   private UserDao userDao;
+
+   private Client client;
+
+   @Before
+   public void createClient() {
+      client = ClientBuilder.newBuilder().build();
```

Comment:
Extract the client attribute as well as these two methods to `io.lumeer.remote.rest.ServiceIntegrationTestBase` abstract class which extends `IntegrationTestBase`. And extend this class in both new service test classes. You will also need to remove these methods and attribute from `ViewServiceIntegrationTest`.